waveform_editor/gui/plotter_edit.py

```diff
@@ -39,6 +42,13 @@ def update_plot(self):
         if self._update_plot_from_drag:
             return  # Skip update triggered from a drag-and-drop
 
+        if isinstance(self.plotted_waveform, DerivedWaveform):
+            try:
+                self.pane.object = self.main_curve()
+            except Exception as e:
+                self.editor.create_error_alert(e)
+            return
```

Comment:
Let's just install a generic error handler instead of these.
See https://stackoverflow.com/questions/57339953/raise-errors-and-display-traceback-messages-from-panel-dashboards#72167334